sources/packages/web/src/types/AppRoutes.ts

```diff
@@ -27,6 +27,7 @@ export enum AppRoutes {
   ManageInstitutionDesignation = "manage-desgination",
   InstitutionManageUsers = "manage-users",
   DisabledUser = "login/disabled-user",
+  UnknownUser = "login/unknown-user",
```

Comment:
"UnknownUser" is a bit tricky IMO. We know the user, the user is valid, the user belongs to that institution and the user knows it.
I would change it to something that indicates that the user is valid but is not allowed to access the system and he needs permission from the administrator.
The variable name could be something like:
- userNotAllowedInApplication
- userNotInManageUsers
It is not a blocker if the Team is ok with it.

Reply:
mmm, IMO, `unknown-user` seems more close, like a valid user, which is unknown to our application

Reply:
It is fine for me 😉